packages/core/src/__tests__/Tooltip.test.js

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { shallow } from 'enzyme';
-import Tooltip from '../Tooltip';
 
-it('renders without crashing', () => {
-    const div = document.createElement('div');
-    const element = <Tooltip>Content</Tooltip>;
+import Tooltip, { PureTooltip } from '../Tooltip';
 
-    ReactDOM.render(element, div);
+describe('<mixed(Tooltip)>', () => {
```

Comment:
`<mixed(Tooltip)>` sounds weird, since we do not have any `mixed` HOC.

Or just leave it `<Tooltip>` is enough to understand?

Reply:
But `<Tooltip>` is also confusing with `Pure <Tooltip>`.
How about `<Tooltip> with mixins`

Reply:
Sounds good